travis_script.sh

```diff
@@ -8,6 +8,7 @@ declare -a  PROJECT_NAMES=(
     "platform_design"
     "platform_view_swift" \
     "provider_counter" \
+    "provider_shopper" \
```

Comment:
Thanks for remembering to add it here. Sine this PR was started, we also began standardizing analysis_options.yaml files as well. Can you copy in the file from one of the other samples?